robshape/blockval
# Added base contracts and updated build process

# Change Description
Added base for contracts, representing a simplified election, and updated how contracts are built.

# How to Verify
Run `npm install` and `npm run build`. Contract JSON outputs, including bytecode, should be written to `dist/ethereum`.

# Definition of Done
~~- [ ] Automated tests~~
